docs: add note about CCloud default for processing log (DOCS-13007) (#8886)
confluentinc/ksql: docs/reference/processing-log.md

setting the ksqlDB property `ksql.logging.processing.rows.include` to
`true`.

!!! important
    In {{ site.ccloud }}, `ksql.logging.processing.rows.include` is set
    to `true`, so the default behavior is to include row data in the
    processing log. Contact support to disable this default setting.

> If you set `ksql.logging.processing.rows.include` to `true`, ensure that the

Reviewer: ```suggestion
When `ksql.logging.processing.rows.include` is set to `true`, ensure that the
```
